### PR TITLE
chore: change text of website regarding header

### PR DESCRIPTION
**Pull request by cdrage**
chore: change text of website regarding header

### What does this PR do?

Changes the text as per penpot for the index page, removing the uneeded
"- Podman Desktop" as well as simplyfing the description

### What issues does this PR fix or reference?

Part of https://github.com/containers/podman-desktop-internal/issues/376

### How to test this PR?

Preview website

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

**Comment by benoitf** [comment]
Changes from Argos: (right is the PR/change)

image

**Comment by benoitf** [comment]
I guess it's @Firewall & @slemeur that need to approve there :-)

**Review by gastoner — APPROVED** [review]

**Review by Firewall — APPROVED** [review]
Change in text makes sense to me. More concise, less repitition 👍

**Comment by slemeur** [comment]
Before merging - how this affect SEO?
Having a description like this helped

File: website/src/pages/index.tsx [hunk 1]
```diff
@@ -105,12 +105,11 @@ function Hero(): JSX.Element {
...
           <div className="bg-white/30 dark:bg-transparent">
             <h1 className="title-font sm:text-4xl text-3xl lg:text-6xl mb-8 font-medium text-gray-900 dark:text-white leading-[1.2]">
-              Best Free &amp; Open Source Tool for Containers &amp; Kubernetes - Podman Desktop
+              Best Free &amp; Open Source Tool for Containers &amp; Kubernetes
             </h1>
             <p className="text-base md:text-lg">
-              Podman Desktop is the best free and open source tool to work with Containers and Kubernetes for
-              developers. Get an intuitive and user-friendly interface to effortlessly bridge the gap between local
-              development and Kubernetes clusters. Build, manage, and deploy everywhere with ease from a single tool.
+              Effortlessly bridge the gap between local development and Kubernetes clusters. Build, manage, and deploy
```
Comment (slemeur) [inline website/src/pages/index.tsx]: I think it is important to callout the target audience we have.

File: website/src/pages/index.tsx [hunk 2]
```diff
@@ -105,12 +105,11 @@ function Hero(): JSX.Element {
         <div className="text-center lg:w-2/3 w-full bg-hero-pattern bg-no-repeat bg-center">
           <div className="bg-white/30 dark:bg-transparent">
             <h1 className="title-font sm:text-4xl text-3xl lg:text-6xl mb-8 font-medium text-gray-900 dark:text-white leading-[1.2]">
-              Best Free &amp; Open Source Tool for Containers &amp; Kubernetes - Podman Desktop
+              Best Free &amp; Open Source Tool for Containers &amp; Kubernetes
```
Comment (slemeur) [inline website/src/pages/index.tsx:108]: Fine for this one

**Review by slemeur — CHANGES_REQUESTED** [review]

File: website/src/pages/index.tsx (hunk 1 above)
Comment (cdrage) [reply website/src/pages/index.tsx]: I have reverted this back to the original one.

**Comment by cdrage** [comment]
@slemeur Reverted it back to the original text, but removed the part that said `- Podman Desktop`.

Please re-review :)

**Comment by MariaLeonova** [comment]
@cdrage do you think there's a way to make the title text 2 lines, not 3? Maybe adjusting size in https://github.com/podman-desktop/podman-desktop/pull/10035 will help?

**Comment by cdrage** [comment]
> @cdrage do you think there's a way to make the title text 2 lines, not 3? Maybe adjusting size in #10035 will help?

There's a lot of text, even adjusting the font size and line height doesn't help :(

@slemeur do you have suggestions on something smaller and SEO friendly for the text?

**Comment by slemeur** [comment]
What do you think about this description:
"Podman Desktop: the best free and open-source tool for developers to work with containers and Kubernetes. Simplify container management, streamline Kubernetes workflows, and transition from local development to production with ease."

**Comment by cdrage** [comment]
> What do you think about this description: "Podman Desktop: the best free and open-source tool for developers to work with containers and Kubernetes. Simplify container management, streamline Kubernetes workflows, and transition from local development to production with ease."

I feel like we repeat "Podman Desktop" in sentences a lot, would:

`The best free and open-source tool for developers to work with containers and Kubernetes. Simplify container management, streamline Kubernetes workflows, and transition from local development to production with ease.`

Be okay?

**Comment by slemeur** [comment]
OK. LGTM

**Comment by cdrage** [comment]
Updated @slemeur

**Comment by slemeur** [comment]
LGTM

**Review by slemeur — APPROVED** [review]
open source instead of "open-source" would be better.

**Comment by cdrage** [comment]
> open source instead of "open-source" would be better.

Changed to open source! Going to enable auto-merge now this PR has multiple reviews.

**Pull request merged**